airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/RelationalDbQueryUtils.java

```diff
@@ -29,11 +29,28 @@ public static String enquoteIdentifierList(final List<String> identifiers, final
     return joiner.toString();
   }
 
-  public static String getFullTableName(final String nameSpace, final String tableName, final String quoteString) {
+  /**
+   * @return fully qualified table name with the schema (if a schema exists) in quotes.
+   */
+  public static String getFullyQualifiedTableNameWithQuoting(final String nameSpace, final String tableName, final String quoteString) {
     return (nameSpace == null || nameSpace.isEmpty() ? getIdentifierWithQuoting(tableName, quoteString)
         : getIdentifierWithQuoting(nameSpace, quoteString) + "." + getIdentifierWithQuoting(tableName, quoteString));
   }
 
+  /**
+   * @return the input identifier with quotes.
+   */
+  public static String enquoteIdentifier(final String identifier, final String quoteString) {
+    return quoteString + identifier + quoteString;
+  }
+
+  /**
```

Comment:
_SUPER_ minor nit, can we have this method grouped with `getFullyQualifiedTabledNameWithQuoting` similar to how it was originally grouped? The rationale behind this nit is because it's easier for someone quickly reviewing this file to know the non-quoted version exists if it's logically grouped together

Reply:
Done